apps/server/src/modules/files-storage/entity/filerecord.entity.spec.ts

```diff
@@ -113,4 +118,21 @@ describe('FileRecord Entity', () => {
 			expect(securityCheck.requestToken).toEqual(securityCheck.requestToken);
 		});
 	});
+
+	describe('setName is called', () => {
+		const setup = () => {
```

Comment:
Maybe it looks overkill but to avoid miss understanding it is possible to add a additional description block between lines 122 and 123 place with the when senario.

Is it allowed to add empty names?

Reply:
setName throws error on empty string and two scenario describe blocks are added to the test.